.github/workflows/cron-licenses.yml

```diff
@@ -3,6 +3,7 @@ name: cron-licenses
 on:
   schedule:
     - cron: "7 0 * * 1" # every Monday at 00:07 UTC
+  workflow_dispatch:
```

Comment:
What does this do?

Reply:
This allows us to run this specific cron on demand in case it needs to be run for whatever reason manually without having to update cron. An example of the case is right before a release needing to ensure that the crons are run.

Reply:
Ah yes, makes senst to have it on all crons.